setup/tue-functions.bash

```diff
@@ -22,6 +22,37 @@ function _list_subdirs
     done
 }
 
+# ----------------------------------------------------------------------------------------------------
+#                                       APT MIRROR SELECTION
+# ----------------------------------------------------------------------------------------------------
+
+function tue-apt-select-mirror
+{
+    # Function to set the fastest APT mirror
+    # It uses apt-select to generate a new sources.list, based on the current one.
+    # All Arguments to this functions are passed on to apt-select, so check the
+    # apt-select documentation for all options.
+	hash pip2 2> /dev/null|| sudo apt-get install python-pip
+	hash apt-select 2> /dev/null|| pip2 install --user apt-select
+
+	local mem_pwd=$PWD
+    # shellcheck disable=SC2164
+	cd /tmp
+	local err_code
+	apt-select "$@" 1> /dev/null
+	err_code=$?
+	if [ $err_code == 4 ]
+	then
+		echo -e "Fastest apt mirror is the current one"
+	else
```

Comment:
This has chances of failing. If the command has a different error code, `sources.list` will not be created and the `else` block will start spitting out errors.